[2021-07-08 13:26:43,657] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.hook_users 2021-07-08T13:26:37.545219+00:00 [queued]>
[2021-07-08 13:26:43,685] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.hook_users 2021-07-08T13:26:37.545219+00:00 [queued]>
[2021-07-08 13:26:43,688] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-08 13:26:43,689] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-07-08 13:26:43,689] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-08 13:26:43,705] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): hook_users> on 2021-07-08T13:26:37.545219+00:00
[2021-07-08 13:26:43,717] {standard_task_runner.py:52} INFO - Started process 18761 to run task
[2021-07-08 13:26:43,732] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'bluepi_dag', 'hook_users', '2021-07-08T13:26:37.545219+00:00', '--job-id', '145', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/bluepi_dag.py', '--cfg-path', '/tmp/tmp95guou8m', '--error-file', '/tmp/tmpajqltr7f']
[2021-07-08 13:26:43,742] {standard_task_runner.py:77} INFO - Job 145: Subtask hook_users
[2021-07-08 13:26:43,806] {logging_mixin.py:104} INFO - Running <TaskInstance: bluepi_dag.hook_users 2021-07-08T13:26:37.545219+00:00 [running]> on host 3cf0296804bd
[2021-07-08 13:26:43,880] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=natchanon
AIRFLOW_CTX_DAG_ID=bluepi_dag
AIRFLOW_CTX_TASK_ID=hook_users
AIRFLOW_CTX_EXECUTION_DATE=2021-07-08T13:26:37.545219+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-08T13:26:37.545219+00:00
[2021-07-08 13:26:43,893] {base.py:78} INFO - Using connection to: id: my_postgres_conn. Host: 35.247.174.171, Port: 5432, Schema: postgres, Login: exam, Password: ***, extra: {}
[2021-07-08 13:26:45,496] {logging_mixin.py:104} INFO - (datetime.datetime(2020, 2, 16, 18, 33, 57, 796067, tzinfo=datetime.timezone.utc), datetime.datetime(2020, 2, 16, 18, 33, 57, 796067, tzinfo=datetime.timezone.utc), '55514cf0-3026-404f-8ea3-f41b00bdf6b5', 'John', 'Henry')
[2021-07-08 13:26:45,496] {logging_mixin.py:104} INFO - (datetime.datetime(2020, 2, 16, 18, 33, 57, 796067, tzinfo=datetime.timezone.utc), datetime.datetime(2020, 2, 16, 18, 33, 57, 796067, tzinfo=datetime.timezone.utc), 'd0e73a35-ff6a-4f64-89b4-ed2b813782a3', 'สมบูรณ์', 'รุ้งแก้ว')
[2021-07-08 13:26:45,497] {python.py:151} INFO - Done. Returned value was: None
[2021-07-08 13:26:45,510] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=bluepi_dag, task_id=hook_users, execution_date=20210708T132637, start_date=20210708T132643, end_date=20210708T132645
[2021-07-08 13:26:45,545] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-07-08 13:26:45,589] {local_task_job.py:151} INFO - Task exited with return code 0
